package.json

```diff
@@ -50,20 +43,25 @@
 		]
 	},
 	"devDependencies": {
-		"@types/react-router-dom": "^5.1.5",
-		"@types/styled-components": "^5.1.0",
-		"@typescript-eslint/eslint-plugin": "^2.30.0",
-		"@typescript-eslint/parser": "^2.30.0",
-		"eslint": "^6.8.0",
-		"eslint-config-airbnb": "^18.1.0",
-		"eslint-config-prettier": "^6.11.0",
-		"eslint-import-resolver-typescript": "^2.0.0",
-		"eslint-plugin-import": "^2.20.1",
-		"eslint-plugin-jsx-a11y": "^6.2.3",
-		"eslint-plugin-prettier": "^3.1.3",
-		"eslint-plugin-react": "^7.19.0",
-		"eslint-plugin-react-hooks": "^2.5.0",
-		"gh-pages": "^3.1.0",
-		"prettier": "^2.0.5"
+		"@types/react-router-dom": "5.1.5",
+		"@types/styled-components": "5.1.0",
+		"@types/jest": "24.9.1",
+		"@types/node": "12.12.62",
+		"@types/react": "16.9.49",
+		"@types/react-addons-css-transition-group": "15.0.5",
+		"@types/react-dom": "16.9.8",
```

Comment:
i put all the @types dependencies on devDependencies and fixed the version to avoid "surprises".